bindata/network/ovn-kubernetes/common/002-rbac-node.yaml

```diff
@@ -138,6 +138,7 @@ rules:
   resources:
   - adminpolicybasedexternalroutes
   - egressfirewalls
+  - egressfirewalls/status
```

Comment:
Why did you need this and not  `adminpolicybasedexternalroutes/status` as well?

Reply:
I actually need, but it should've already been here, because node updates status already :confused:

Reply:
created a separate PR for that, since it is a different problem and should be backported to 4.14 https://github.com/openshift/cluster-network-operator/pull/2139